src/views/wiki/edition/utils/FormInput.vue

```diff
@@ -89,11 +101,25 @@
 </template>
 
 <script>
+
+    import Datepicker from 'vuejs-datepicker'
+    import { fr, en, es, ca, de, it, eu } from 'vuejs-datepicker/dist/locale'
+
     import { requireDocumentProperty, requireFieldProperty } from '@/js/properties-mixins'
     import InputConditionsLevels from './InputConditionsLevels'
 
+    // note that eu is missing. Sorry euskara...
```

Comment:
have you studied how locales are built-in? it might be quite easy to add euskara and propose a patch?

Reply:
No I didn't. It may be quit easy.

But considering that datepicker's translation is an evolution comparing to V6, and that euskara is a low c2c audience, it's not a big prio :)

Reply:
Could you open a issue on c2cui anyway, to keep track of it?

Reply:
#128 :)